site/public/css/submitbox.css

```diff
@@ -13,6 +13,15 @@
     margin-left: auto;
 }
 
+#error-message {
+    text-align: center;
+    margin-left: auto;
+}
+
+#error-message > h1 {
+    color: red;
```

Comment:
It would be better to use one of our existing colors from `colors.css`.  A good choice might be the same color used for the red warning text shown above the submit box in your screenshot.